tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "noEmit": true,
-    "jsx": "react",
```

Comment:
Op "preserve" werkt `npm run test` niet.

Reply:
wss maken we dan best een tsconfig.test.json die extend van de tsconfig.json
en dan referencen we die in jest.config.js

Reply:
react-scripts gaat deze altijd automatisch terugzetten naar preserve 🤷‍♂️